test/second_q/operators/test_bosonic_op.py

```diff
@@ -198,25 +198,25 @@ def test_index_order(self):
             self.assertEqual(bos_op, targ)
 
         with self.subTest("index order + simplify"):
-            orig = BosonicOp({"+_1 -_0 +_0 -_0": 1, "-_0 +_1": 2})
+            orig = BosonicOp({"+_1 -_0": 1, "-_0 +_1": 2})
             bos_op = orig.index_order().simplify()
-            targ = BosonicOp({"+_1 -_0": 4})
+            targ = BosonicOp({"-_0 +_1": 3})
             self.assertEqual(bos_op, targ)
 
     def test_simplify(self):
         """Test simplify
         This test method tries to simplify the operator label
         """
         with self.subTest("simplify integer"):
-            bos_op = BosonicOp({"+_0 -_0": 1, "+_0 -_0 +_0": 1}, num_modes=1)
+            bos_op = BosonicOp({"+_0 -_0": 1, "-_0 +_0": 1}, num_modes=1)
             simplified_op = bos_op.simplify()
-            targ = BosonicOp({"+_0 -_0": 1, "+_0": 2}, num_modes=1)
+            targ = BosonicOp({"+_0 -_0": 1, "-_0 +_0": 1}, num_modes=1)
             self.assertEqual(simplified_op, targ)
 
         with self.subTest("simplify complex"):
-            bos_op = BosonicOp({"+_0": 1, "+_0 -_0 +_0": 1j}, num_modes=1)
+            bos_op = BosonicOp({"+_0 -_0": 1}) + BosonicOp({"+_0 -_0": 1j})
             simplified_op = bos_op.simplify()
-            targ = BosonicOp({"+_0": (1 + 2j)}, num_modes=1)
+            targ = BosonicOp({"+_0 -_0": (1 + 1j)}, num_modes=1)
             self.assertEqual(simplified_op, targ)
```

Comment:
These tests no longer really make sense, because in the first case nothing changes (although we could ofcourse test for that) and in the second case the addition already does the correct logic.

Reply:
I would remove only the second one. As you say, I think that it's useful to test that nothing changes